core/domain/suggestion_registry.py

```diff
@@ -508,6 +510,7 @@ def __init__( # pylint: disable=super-init-not-called
         self.change.question_dict['question_state_data_schema_version'] = (
             feconf.CURRENT_STATE_SCHEMA_VERSION)
         self.score_category = score_category
+        self.skill_difficulty = constants.DEFAULT_SKILL_DIFFICULTY
```

Comment:
This won't align with the BaseSuggestion object, right? Do we need this data to create the suggestion model? (Can we do it the way we have handled commit_message in Content suggestion accept method around line 343?)